### PR TITLE
417, 418: modified transform to work with json from scancode and renamed configuration key for transform

### PR DESCRIPTION
**Pull request by srthkdb**
1. Processes only the data inside "files"
2. Automatically renames "path" to "about_resource"
3. Written tests

### CODE REVIEW

**Comment by chinyeungli** [comment]
@srthkdb Can you try to fix the failing tests?

**Comment by srthkdb** [comment]
@chinyeungli  Fixed!

**Comment by srthkdb** [comment]
I have also renamed column_renamings, required_columns, and column_filters to field_renamings, required_fields, and field_filters respectively.

File: src/attributecode/transform.py [hunk 1]
```diff
@@ -129,77 +129,24 @@ def transform_json(data, transformer):
...
-                element, err = process_json_keys(item, renamings, transformer)
-                for e in element:
-                    new_data.append(e)
-                for e in err:
-                    errors.append(e)
+        #only takes data inside "files"
+        data = data["files"]
+        #automatically renames path to about_resource
+        if("path" not in renamings.keys()):
+            renamings["path"] = "about_resource"
```
Comment (chinyeungli) [inline src/attributecode/transform.py]: @srthkdb I tend not to do this automatically in code side as, who knows, scancode may change the variable name in the future (thou it's very less likely), or AboutCode TK may change the "about_resource" to "path".

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/sarthak/gsoc/aboutcode-toolkit/local/bin/python2.7"
```
Comment (chinyeungli) [inline .vscode/settings.json]: Perhaps this should be removed and shouldn't be committed

File: src/attributecode/transform.py (hunk 1 above)
Comment (srthkdb) [reply src/attributecode/transform.py]: Removed it!

File: .vscode/settings.json (hunk 2 above)
Comment (srthkdb) [reply .vscode/settings.json]: I deleted the folder, I don't know why is this still showing up.

**Pull request merged**